### ISSUE TITLE
Key & System Maintenance features

### ISSUE THREAD

**Issue by Forcimeller** [Forcimeller/Key-Management-System#4]
Insert is already done in another Ticket, now we need to permit updating (So replacing) and deleting keys.

**Comment by Forcimeller** [comment]
Not just Key maintenance, system maintenance and diagnostics. 

this means the implementation of these:

`--seeKeys <password>` requires a password and will display all keys
`--seeLogs <password>` Displays all logs
`--rmKey <password> <key name>` removes the specified key
`--updKey <password> <key name> <path to file>` replaces an existing key

**Comment by Forcimeller** [comment]
Each and every time I have to fiddle with MongoDB, it's so much BS

I had an error "what (): Expected element type k_string" I spent 4 hours thinking this was an issue with my query, when In fact, it was an issue with my struct generator. 

It was struggling to process the datetime, so now I had to make an entirely new subroutine for casting from the datetime stored in the database to another thing, to another thing, to another thing to an object stream to a string. what a complete mess. I have no idea how it does it either. OR WHY... Damn c++ 🙄 This would never be an issue in Java.

Up next? `--seeKeys`. This should be easier as we are only dealing with strings.
